test/BuiltinsWithGenericPointer/issue-1343.cl

```diff
@@ -0,0 +1,12 @@
+// RUN: clspv --cl-std=CLC++ --inline-entry-points %s --print-before=annotation-to-metadata --output-format=ll -O3 -o %t.ll
+// RUN: FileCheck %s < %t.ll
+
+// CHECK: define dso_local spir_func float @_Z22contains_frexp_in_namefPU3AS4i(float %0, ptr addrspace(4) %1)
+// CHECK: define dso_local spir_func void @_Z12vstore_half5Dv2_fjPU3AS4Dh(<2 x float> %0, i32 %1, ptr addrspace(4) %2)
+// CHECK: define dso_local spir_func void @_Z13xvstore_half2Dv2_fjPU3AS4Dh(<2 x float> %0, i32 %1, ptr addrspace(4) %2)
+
```

Comment:
The way it is right now is not failing if I do not use the version of `BuiltinWithGenericPointer` in this CL.
```suggestion
// RUN: clspv --cl-std=CLC++ --inline-entry-points %s --print-before=annotation-to-metadata -o %t.spv &> %t.ll
// RUN: FileCheck %s < %t.ll
// RUN: spirv-val %t.spv

// CHECK: define dso_local spir_func float @_Z22contains_frexp_in_namefPU3AS4i(float %0, ptr addrspace(4) %1)
// CHECK: define dso_local spir_func void @_Z12vstore_half5Dv2_fjPU3AS4Dh(<2 x float> %0, i32 %1, ptr addrspace(4) %2)
// CHECK: define dso_local spir_func void @_Z13xvstore_half2Dv2_fjPU3AS4Dh(<2 x float> %0, i32 %1, ptr addrspace(4) %2)

void kernel foo() {}
```

Reply:
Ah ok... I'm pretty sure I tested it but maybe I got something wrong. I'll commit your suggestion :)